static/js/pages/superuser.js

```diff
@@ -111,10 +112,22 @@
                                                            ['name', 'email'], []);
       };
 
+    var caclulateRegistryStorage = function () {
+      if (!Features.QUOTA_MANAGEMENT || !$scope.organizations) {
+        return;
+      }
+      let total = 0;
+      $scope.organizations.forEach(function (obj){
```

Comment:
i.e have a superuser specific endpoint that just does this repositorysize table aggregation.

Reply:
We have the Quota Consumption data for every org under `quota_report` key. So, instead of making another API call, we are using the data the `/organizations` endpoint is fetching.

Reply:
Ah ok. I guess that panel is already fetching the complete list of organization.